src/main/g8/$name__norm$-impl/src/main/java/$package$/impl/$name__Camel$Aggregate.java

```diff
@@ -0,0 +1,124 @@
+package $package$.impl;
+
+import akka.cluster.sharding.typed.javadsl.EntityContext;
+import akka.cluster.sharding.typed.javadsl.EntityTypeKey;
+import akka.persistence.typed.PersistenceId;
+import akka.persistence.typed.javadsl.*;
+import com.lightbend.lagom.javadsl.persistence.AkkaTaggerAdapter;
+import $package$.impl.$name;format="Camel"$Command.Hello;
+import $package$.impl.$name;format="Camel"$Command.UseGreetingMessage;
+import $package$.impl.$name;format="Camel"$Event.GreetingMessageChanged;
+
+import java.util.Set;
+
+/**
+ * This is an event sourced aggregate. It has a state, {@link $name;format="Camel"$State}, which
+ * stores what the greeting should be (eg, "Hello").
+ * <p>
+ * Event sourced aggregate are interacted with by sending them commands. This
+ * aggregate supports two commands, a {@link UseGreetingMessage} command, which is
+ * used to change the greeting, and a {@link Hello} command, which is a read
+ * only command which returns a greeting to the name specified by the command.
+ * <p>
+ * Commands may emit events, and it's the events that get persisted.
+ * Each event will have an event handler registered for it, and an
+ * event handler simply applies an event to the current state. This will be done
+ * when the event is first created, and it will also be done when the entity is
+ * loaded from the database - each event will be replayed to recreate the state
+ * of the aggregate.
+ * <p>
+ * This aggregate defines one event, the {@link GreetingMessageChanged} event,
+ * which is emitted when a {@link UseGreetingMessage} command is received.
+ */
+public class $name;format="Camel"$Aggregate extends EventSourcedBehaviorWithEnforcedReplies<$name;format="Camel"$Command, $name;format="Camel"$Event, $name;format="Camel"$State> {
+
+    public static EntityTypeKey<$name;format="Camel"$Command> ENTITY_TYPE_KEY =
+        EntityTypeKey
+            .create($name;format="Camel"$Command.class, "$name;format="Camel"$Aggregate");
+
+
+    final private EntityContext<$name;format="Camel"$Command> entityContext;
+    final private String entityId;
+
+    $name;format="Camel"$Aggregate(EntityContext<$name;format="Camel"$Command> entityContext) {
+        super(
+            PersistenceId.of(
+                entityContext.getEntityTypeKey().name(),
+                entityContext.getEntityId(),
+                // The separator is an optional field. If you want to produce
+                // events compatible with Lagom read-side processors, then the
+                // separator must be an empty String.
+                ""
```

Comment:
Better not reproduce that pattern in new projects. We should only recommend "" for existing projects migrating to Akka Persistence Typed